config/sync/field.field.paragraph.components.field_p_c_component.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
     - paragraphs.paragraphs_type.image_paragraph
     - paragraphs.paragraphs_type.rich_text
     - paragraphs.paragraphs_type.slideshow
+    - paragraphs.paragraphs_type.webform_embed
```

Comment:
This should only be added to the basic page for the moment

Reply:
done